requirements-dev.txt

```diff
@@ -14,14 +14,14 @@ scipy
 s3fs>=2021.8
 pytest
 pytest-benchmark
-coverage<5.0
+coverage
```

Comment:
We should also change the reqs in other env files.

Reply:
@Garra1980, what about my comment above? The env files should be consistent as much as possible. Would you like to resolve this as part of a separate issue?

Reply:
> @Garra1980, what about my comment above? The env files should be consistent as much as possible. Would you like to resolve this as part of a separate issue?

Yes, I'd like to do this separately

Reply:
Most of our tests in CI are ran under `environment-dev.yml` environment. So, since that file is unchanged in the PR we can't determine, whether it affects passing of tests or not (maybe these pins were done for a certain reason and reverting them would break tests)
https://github.com/modin-project/modin/blob/81cb913b16d4368915f7c3ee2d66d8eba4e750d3/.github/workflows/ci.yml#L439

Reply:
Correct, @prutskov did local testing in requirements-dev env and all was fine. thought we already mentioned this...

Reply:
@Garra1980, please add a release note in `docs/release_notes/release_notes-0.15.0.rst`. Meanwhile, could you submit a PR where the reqs are relaxed in other env files - `environment-dev.yml`, `requirements-no-engine.yml`, `env_omnisci.yml`, in order to see tests pass?

Reply:
RN updated

Reply:
#4380 created

Reply:
Good, thanks!